src/programs/tokenMetadata/instructions/createMintNewEditionFromMasterEditionViaVaultProxyInstructionWithSigners.ts

```diff
@@ -0,0 +1,75 @@
+import BN from 'bn.js';
+import { PublicKey } from '@solana/web3.js';
+import { createMintNewEditionFromMasterEditionViaVaultProxyInstruction } from '@metaplex-foundation/mpl-token-metadata';
+import { Signer } from '@/types';
+import { InstructionWithSigners } from '@/utils';
+
+export interface CreateMintNewEditionFromMasterEditionViaVaultProxyInstructionWithSignersParams {
```

Comment:
Is this supposed to be extended by package users? Otherwise use `type` here instead of `interface` for consistency.